File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1052,32 +888,24 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Token-based authentication
 
-The Blink HTML converter supports token-based authentication by using the HTTP request headers. The token values will be send to web server when the HTML page is requested. Refer to the following code snippet.
+The Blink HTML converter supports token-based authentication by [HttpRequestHeaderCollection](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.HttpRequestHeaderCollection.html) class using the [HttpRequestHeaders](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_HttpRequestHeaders) property. The token values will be send to web server when the HTML page is requested. Refer to the following code snippet.
```

Comment:
The Blink HTML converter supports token-based authentication by using the [HttpRequestHeaders]() property in [BlinkConverterSettings]() class.

Reply:
Modified